eng/CredScanSuppression.json

```diff
@@ -43,6 +43,10 @@
             "placeholder": "OAuth2ClientCredential",
             "_justification": "Javadoc in azure-resourcemanager-datafactory"
         },
+        {
+            "placeholder": "CacheableManageIdentityCredential",
```

Comment:
Why is this needed here? If it is a false positive we should try to reuse an existing placeholder or add this to the existing list of placeholders if we cannot change it for some reason.

Reply:
I don't believe this should be an error for credscan. I have contacted credscan team for support. If this is an urgent change, could you add this to the placeholder list above and cut an issue to me? I will follow up.

Reply:
@backwind1233 Since the placeholder is a little hard to be specific, I think we'd better to follow credscan team suggestion to use hash suppression rule. 
```
{
  "tool": "Credential Scanner",
  "suppressions": [
    {
      "hash": "fxfINtmkGJO0+DFqu2DFNX0Q9y4JC5GhTWGK2Uzvn1I=",
      "_justification": "Java class in spring-cloud-azure-service."
    }
  ]
}
```

@weshaggard I think we'd better to publish the cred scan output to our pipeline, so people can fetch the hash value themselves.

Reply:
I just responded to the email asking some more questions. The biggest reason I don't like these types of suppressions is because they get invalidated anytime the file changes and so we have to change them every time we change the file. If there isn't any other good way then we can do this but I sort of like the placeholder value instead.

Reply:
We have an email thread for the issue and how to suppress this. It is actually a false positive case. We do need to suppress before credscan team fixed the issue. We currently do not support the exact secret match for placeholder. The best way is to suppress using hash value. Please use the block I share above and have better justification for future removal.

Reply:
Thank you @sima-zhu , we have refactored our code and we don't have the class `CacheableManageIdentityCredential` any more. So the pipeline will not block by credscan.😀